drivers/sdhc/rcar_mmc.c

```diff
@@ -1762,6 +1763,15 @@ static int rcar_mmc_init(const struct device *dev)
 		dev->name, data->ver);
 
 	return 0;
+
+_disable_clk:
```

Comment:
This is not (probably) codified in the Zephyr coding style, but it is not customary to use underscores in the label names.  Your code will differ from the surrounding code. So I am strictly suggesting to rename your labels.

Reply:
changed names